### PR TITLE
feat(Paddle frontend API): added t tensor method and test

### LINKED ISSUE #26056: t

### PR DESCRIPTION
**Pull request by Kartik0605**
# PR Description
This PR will add t paddle frontend API method and its respective test.

## Related Issue

Close #26056 

## Checklist

- [X] Did you add a function?
- [X] Did you add the tests?
- [X] Did you run your tests and are your tests passing?
- [X] Did pre-commit not fail on any check?
- [X] Did you follow the steps we provided?

### CODE REVIEW

File: ivy/functional/frontends/paddle/tensor/tensor.py [hunk 1]
```diff
@@ -796,6 +796,11 @@ def less_equal(self, y, name=None):
     def real(self, name=None):
         return paddle_frontend.real(self._ivy_array)
 
+    @with_unsupported_dtypes({"2.5.1 and below": ("float16", "bfloat16")}, "paddle")
+    def t(self, name=None):
+        axes = list(range(len(self._ivy_array.shape)))[::-1]
```
Comment (hello-fri-end) [inline ivy/functional/frontends/paddle/tensor/tensor.py:801]:
```suggestion
        axes = list(range(len(self.ivy_array.shape)))[::-1]
```
We shouldn't access a private attribute :)

File: ivy/functional/frontends/paddle/tensor/tensor.py [hunk 2]
```diff
@@ -796,6 +796,11 @@ def less_equal(self, y, name=None):
...
+        return ivy.permute_dims(self._ivy_array, axes=axes)
```
Comment (hello-fri-end) [inline ivy/functional/frontends/paddle/tensor/tensor.py:802]:
```suggestion
        return ivy.permute_dims(self.ivy_array, axes=axes)
```

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py [hunk 3]
```diff
@@ -4006,6 +4006,42 @@ def test_paddle_tensor_subtract_(
     )
 
 
+# t
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="paddle.to_tensor",
+    method_name="t",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```
Comment (hello-fri-end) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py:4015]:
```suggestion
        available_dtypes=helpers.get_dtypes("valid"),
```
This should always be set to `valid` - if the function only supports floating-point data, then that's what valid will return :)

**Review by hello-fri-end — CHANGES_REQUESTED** [review]
Hey @Kartik0605 , thanks for creating this PR - it's already in very good shape. Just left a few minor comments, let's resolve those after which this is good to merge:)

**Comment by Kartik0605** [comment]
@hello-fri-end i have updated the changes that u requested, please take a look again.

**Review by hello-fri-end — APPROVED** [review]
Looks like there are no failures on the CI. Merging this now, thanks for the contribution @Kartik0605

**Pull request merged**